tests/rpc.test.js

```diff
@@ -0,0 +1,128 @@
+import React from "react";
+import { render, act, screen, waitFor } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import RPC from "../components/RPC-Connection";
+import { ApiPromise, WsProvider } from "@polkadot/api";
+
+// Set max test duration before failing (2 min)
+jest.setTimeout(120000);
+
+test("Retrieves and applies a 'const' RPC value", async () => {
+	render(<RPC network="polkadot" path="consts.balances.existentialDeposit" defaultValue={0} />);
+	await waitFor(() => expect(screen.getByText("10000000000")).toBeInTheDocument(), { timeout: 2000 });
+});
```

Comment:
I want to try loading the entire app and write more generic tests that can potentially find errors in the docs, but I will attempt that in another PR.